netbox_custom_objects/models.py

```diff
@@ -214,6 +216,15 @@ class Meta:
     def __str__(self):
         return self.display_name
 
+    def clean(self):
+        super().clean()
+
+        # Enforce max number of COTs that may be created (max_cots)
+        if not self.pk:
+            max_cots = get_plugin_config("netbox_custom_objects", "max_cots")
+            if max_cots and CustomObjectType.objects.count() > max_cots:
+                raise ValidationError(_(f"Maximum number of Custom Object Types ({max_cots}) exceeded"))
```

Comment:
It might be helpful to include a hint that the configuration parameter can be modified to increase this limit.